BoscoTing/TwitchAdvisor
# Feature/count chatroom info

<h1>Description</h1>
1. Use Airflow to track interested channels automatically.
2. Build new functions to show chart that tracking live-streaming channel which is selected by user.
3. Show the historical and streaming charts on UI by using Flask.
<h1>Testing Scope</h1>
1. Ensure the database server is running.
2. Click the default channel to check the historical stats chart.
3. Check if the "Top Live Channels" are updating every minutes.
4. Click the candidate of "Top Live Channels" to check streaming chart.
